### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by don1001** [wisk/medusa#82]
Hi, is this project still in development? or its dead?

**Comment by XVilka** [comment]
@don1001 you can try [radare2](https://github.com/radare/radare2) or [Cutter](https://github.com/radareorg/cutter) instead, since the project looks dead.

**Comment by don1001** [comment]
> @don1001 you can try [radare2](https://github.com/radare/radare2) or [Cutter](https://github.com/radareorg/cutter) instead, since the project looks dead.

Thanks but I don't like Cutter, found this to be MUCH better https://github.com/REDasmOrg/REDasm-Builds
cool project

**Comment by wisk** [comment]
It's not dead, I just need to focus on specific features and drop others.

**Comment by XVilka** [comment]
@wisk have you considered to contribute into Cutter instead? It already supports more features and will be a win-win for everyone.

**Comment by wisk** [comment]
No, I'm not interested on writing code for a GUI. This is, at least for me, the worst and most tedious part of a project.
Besides, I know nothing about radare and its design / API.
My main motivation behind Medusa is learning stuffs (C++, architecture, file executable, ...) and having a tool for specific needs. I don't really care about features and fancy stuffs.
If I need a reverse engineering software with a nice GUI, I use IDA Pro.
